product_supplierinfo_stock_picking_type/models/purchase_order.py

```diff
@@ -7,6 +7,6 @@ class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
 
     @api.onchange("picking_type_id")
-    def onchange_picking_type_id(self):
```

Comment:
I would add a commented note here saying that this name is to avoid conflict with the one in mrp_subcontracting_dropshipping